### ISSUE TITLE
Quote marks in fig caption breaks ptxprint (Linux)

### ISSUE THREAD

**Issue by KimB2017** [sillsdev/ptx2pdf#207]
Using today's repo version of ptxprint in Wasta 18.04. 
I have had to remove quote marks from my captions; ptxprint fails to print otherwise. Below is the log snippet. This happens every time. 

(using pictures from list "/home/kim/Paratext8Projects/RTTT/PrintDraft/tmpPicLi
sts/45ACTRTTT-draft.SFM.piclist")
{Read piclist line: ACT 1.9 |44_ac_01_03_rg.jpg|col|t||Ta bisiobaa, \futurelet 
\next \let \next =“\next Oru ouroepa.\futurelet \next \let \next =”\next |1:9}
(/home/kim/Paratext8Projects/RTTT/PrintDraft/45ACTRTTT-draft.SFM [1
\openout2 = `ptxprint-ACTRTTT.notepages'.

]
! TeX capacity exceeded, sorry [parameter stack size=10000].
\trace #1#2->
             \relax 
\getonep@ram ...2}\trace {g}{getoneparam: #1 | #2}
                                                  \getonepairp@ram #1 ="" \r...

\body ->\x@ \getonep@ram \p@rams \end 
                                      \ifx \p@rams \empty \morep@ramsfalse \...

\next ->\body 
              \let \next \iterate \else \let \next \relax \fi \next 
“->\futurelet \next 
                      \doqtest 
\p@ram ->Ta bisiobaa, “
                         Oru ouroepa.”
...
l.22 \v 9 
          Tuʼuoaroha ii betekaao hiokoʼi opita, oru ouroepa tohapita baire...
 
Here is how much of TeX's memory you used:
 3581 strings out of 495324
 40388 string characters out of 6194384
 126742 words of memory out of 5000000
 7162 multiletter control sequences out of 15000+600000
 4978 words of font info for 25 fonts, out of 8000000 for 9000
 14 hyphenation exceptions out of 32749
 8009i,12n,10001p,472b,149s stack positions out of 30000i,500n,10000p,200000b,80000s
Output written on ptxprint-ACTRTTT.pdf (1 page).

**Comment by davidg-sil** [comment]
@mhosken, this looks like a clash with the USFM-3 parser, do you agree? But why would it be trying to use the USFM-3 parser by the time its got to the caption? Doesn't it turn off?

@KimB2017, if this is the problem, then switching to some form of 'shaped quotes', rather than boring ascii would be a work-around.

**Comment by KimB2017** [comment]
The problem first happened with “shaped” UTF-8 open/close quotes (“ ”). It also happens with straight quotes, if I remember rightly.

**Comment by davidg-sil** [comment]
Hmm... Further investigation, probably not very helpful: "Shaped quotes" whatever the right name for them is,  seem to be a 
significant problem in the piclist input. I'm not sure what's going on.

Two work arrounds got me these, though. I don't actually know if I need both.

![image](https://user-images.githubusercontent.com/58848232/93815642-ff629e80-fc5e-11ea-8259-37d6c1ff1d59.png)

a) Switch to USFM-3 method, at least for the affected line. This may not actually be useful, but it let straight quotes work:
```
HEB 7.1 The Priest was called "Melchisedek"|src="cn01603.jpg" size="col"  ref="7:1" pgpos="tr" 
```
b) Add `mapping=tex-text` to the font definition, enabling TeX-style quotation marks and switch to them:
```
\def\regular{"Tinos:mapping=tex-text"}
``` 
(maybe just doing it for the caption font might be a good idea, in case it breaks other things)

```
HEB 7.1 The Priest was called ``Melchisedek''|src="cn01603.jpg" size="col"  ref="7:1" pgpos="tr" 
```

Hope this helps!

**Comment by KimB2017** [comment]
Fixed for me in 1.1.9. Thanks!
